src/OrchardCore/OrchardCore.FileStorage.Abstractions/FileStoreFileExistsException.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace OrchardCore.FileStorage;
+
+public class FileStoreFileExistsException : Exception
```

Comment:
The new exception type should inherit from `FileStoreException` to not break existing exception handlers.
Additionally I would suggest a shorter name, consistent with the added base type. E.g. `ExistsFileStoreException`.